events/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+from decimal import Decimal
+from datetime import datetime
+
+class Person(models.Model):
```

Comment:
To będzie trzeba w przyszłości zamienić na autoryzację Django, ale na razie zostawiamy